modules/ducktests/tests/ignitetest/services/utils/control_utility.py

```diff
@@ -31,9 +31,17 @@ class ControlUtility:
     """
     BASE_COMMAND = "control.sh"
 
-    def __init__(self, cluster, text_context):
+    def __init__(self, cluster,
+                 key_store_jks: str = None, key_store_pwd: str = "123456",
```

Comment:
Password 123456 is a magic variable. Are there any alternatives? Config, global variable, param, or smth else?
At least setting global variable in run_tests.sh looks better.

Reply:
fixed